### DISCUSSION TITLE
Vue component disappears on re-render

### DISCUSSION THREAD

**Discussion by Keyaku** [livewire/livewire#4487] [7. Bugs (archived)]
I'm running into the same issue from #388 (nearly 3 years old) using v2.8.2.

I need to update a livewire value from a Vue component; the issue comes up right after the value is updated, where all rendered components (not just Vue's) disappear. This is very frustrating, and none of the solutions in that thread seem to be of help.

A shortened version of my code:
```
<x-form.field name="schedule_days">
	<x-jet-label for="schedule_days" value="{{ __('Time Slots') }}" />
	<input disabled type="text" id="schedule_days" wire:model.lazy="schedule_days" value="{{ old('schedule_days', $plugin->schedule_days ?? null) }}"/>
	<template wire:key="vue_datepicker" wire:ignore.self>
		<div class="bg-white p-2 w-full border rounded">
			<v-date-picker v-model="selected.date" mode="dateTime"> {{-- code --}} </v-date-picker>
		</div>
	</template>
</x-form.field>
...
<script>
vue_datepicker = new Vue({
	el: '#vue_form',
	data: {
		selected: {},
		dates: [],
	},
	watch: {
		selected: function(){
			@this.schedule_days = JSON.stringify(this.dates);  // <--- causes re-render and the component disappears
		},
	},
});
</script>

**Comment by joshhanley** [comment]
@Keyaku there is a plugin to get Livewire and Vue running, are you using that? https://github.com/livewire/vue

**Reply by Keyaku** [reply]
Sorry for the delay; a lot has happened before I could tackle this in full force, including me getting covid, finding a workaround for the issue, and a decision to change companies.
To answer your question: I am not using that, but I'll integrate it next time I'll get the chance. In the meantime, this discussion can be closed without further development as I won't be developing on my current work's Laravel application next month.

In any case, thank you for your support!

**Reply by joshhanley** [reply]
No worries! Hope covid wasn't too bad for you and your feeling better 🙂
